src/UserService.Data.Provider.MsSql.Ef/Migrations/20210908225900_DropImageIdColumnOfUserCertificatesTable.cs

```diff
@@ -0,0 +1,22 @@
+﻿using LT.DigitalOffice.UserService.Models.Db;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+using System;
+
+namespace LT.DigitalOffice.UserService.Data.Provider.MsSql.Ef.Migrations
```

Comment:
And another?

Reply:
I did them at different times

Reply:
if all this migration don't contains in master you can do it

Reply:
++